src/Appwrite/Platform/Modules/Functions/Workers/Builds.php

```diff
@@ -1184,8 +1184,7 @@ protected function buildDeployment(
                             'region' => $project->getAttribute('region')
                         ]));
                     } catch (Duplicate $err) {
-                        $rule = $dbForPlatform->getDocument('rules', $ruleId);
-                        $dbForPlatform->updateDocument('rules', $rule->getId(), new Document([
+                        $rule = $dbForPlatform->updateDocument('rules', $rule->getId(), new Document([
```

Comment:
Since we removed getDocument for rule, we cant use $rule here, we need to do $ruleId